content/blog/2022-05-03-e2e-cv-pt1.md

```diff
@@ -0,0 +1,316 @@
+---
+title:
+  'End-to-End Computer Vision API, Part 1: Data Versioning and ML Pipelines'
+date: 2022-05-03
+description: >
+  In most cases, training a well-performing Computer Vision (CV) model is not
+  the hardest part of building a Computer Vision-based system.  The hardest
+  parts are usually about incorporating this model into a maintainable
+  application that runs in a production environment bringing value to the
+  customers and our business.
+
+descriptionLong: |
+
+  _This is the first part of a three-part series of posts._
+
+  Lately, training a well-performing Computer Vision (CV) model in Jupyter Notebooks became fairly straightforward. 
+  You can use pre-trained Deep Learning models and high-level libraries like `fastai`, `keras`, `pytorch-lightning` etc., that abstract away much of the complexity.
+  However, it's still hard to incorporate these models into a
+  maintainable production application in a way that brings value to the
+  customers and business. 
+
+  Below we'll present the tools that naturally integrate with your git repository and makes this part of the process significantly easier.
+picture: 2022-05-03/e2e-cv-pt1-cover.png
+author: alex_kim
+commentsUrl: https://discuss.dvc.org/t/end-to-end-computer-vision/1178
+tags:
+  - Computer Vision
+  - DVC
+  - CML
+  - Studio
+  - CI/CD
+  - Experiment Tracking
+---
+
+### Introduction
+
+In this series of posts, we'll describe an approach that streamlines the three
+phases that almost all CV projects go through:
+
+1. Active exploration or proof-of-concept phase. Here we try many different
+   approaches to data preprocessing, model architectures, and hyperparameter
+   tuning with the goal of finally settling on the most promising combination.
+2. Application development phase. Here we build all the “plumbing” around the
+   model, that is: getting input data, massaging it into the right format,
+   passing it to the model, and, finally, serving the model's output. The goal
+   here is to end up with a version of an end-to-end application that is robust
+   and well-performing enough to be considered for deployment to the production
+   environment.
+3. Production deployment phase. Here we promote a well-tested version of the
+   application from the development environment to the production environment.
```

Comment:
Do we need this numbered list? It doesn't look like you ever return to it, and it doesn't match the 3 parts of the blog post, so I'm not sure why it's needed. You could probably summarize the series as being about taking a model or CV project from exploration to production.

Reply:
rephrased and remove the list